app/deed/view/view.mustache

```diff
@@ -0,0 +1,140 @@
+<div class="alert-inline">
+    <p>This is a legal document — please check all details carefully before signing.</p>
+</div>
+
+<article class="deed-document--bounds">
+    <h2 class="heading-large">Mortgage deed</h2>
+
+    <div>
+        <h2 class="heading-medium">Property:</h2>
+
+        <p itemscope="" itemtype="http://schema.org/PostalAddress">
```

Comment:
Within the deed, I'm not sure we need to do microdata formatting... Although maybe it's nice. Any thoughts from anyone else?